lib/networkx.rb

```diff
@@ -1,7 +1,37 @@
 require 'set'
-
+require 'nmatrix'
+require 'rb_heap'
+require 'securerandom'
 require 'networkx/version'
 require 'networkx/graph'
 require 'networkx/digraph'
 require 'networkx/multigraph'
 require 'networkx/multidigraph'
+require 'networkx/traversals/bfs'
+require 'networkx/traversals/dfs'
+require 'networkx/traversals/edge_dfs'
+require 'networkx/shortest_path/unweighted'
+require 'networkx/to_matrix'
+require 'networkx/shortest_path/dense'
+require 'networkx/shortest_path/weighted'
+require 'networkx/shortest_path/astar'
+require 'networkx/operators/product.rb'
+require 'networkx/operators/binary.rb'
+require 'networkx/operators/unary.rb'
+require 'networkx/operators/all.rb'
+require 'networkx/flow/utils.rb'
+require 'networkx/flow/edmondskarp.rb'
+require 'networkx/flow/shortestaugmentingpath.rb'
+require 'networkx/flow/capacityscaling.rb'
+require 'networkx/flow/preflowpush.rb'
+require 'networkx/auxillary_functions/cliques.rb'
+require 'networkx/auxillary_functions/dag.rb'
+require 'networkx/auxillary_functions/eccentricity.rb'
+require 'networkx/auxillary_functions/mis.rb'
+require 'networkx/auxillary_functions/mst.rb'
+require 'networkx/auxillary_functions/union_find.rb'
+require 'networkx/auxillary_functions/vitality.rb'
+require 'networkx/auxillary_functions/wiener.rb'
+require 'networkx/auxillary_functions/cycles.rb'
+require 'networkx/link_analysis/hits.rb'
+require 'networkx/link_analysis/pagerank.rb'
```

Comment:
`.rb` is not required - you can remove them

Reply:
would it be better if we create a similar file for all sets of algorithms and then add that file here? Like for flow algorithms, we can make a `flow.rb` and then we can add that file here..
It would be easy to manage the requirements, if we do it that way..

Reply:
Yup, that works.